test-cases/upgrades/rolling-upgrade.yaml

```diff
@@ -13,6 +13,7 @@ stress_cmd_complex_prepare: cassandra-stress user no-warmup profile=/tmp/complex
 stress_cmd_complex_verify_read: cassandra-stress user no-warmup profile=/tmp/complex_schema.yaml  ops'(read1=1,read3=1)' cl=ONE n=5000000 -mode cql3 native -rate threads=1000 -pop seq=1..5000000
 stress_cmd_complex_verify_more: cassandra-stress user no-warmup profile=/tmp/complex_schema.yaml  ops'(read1=1,read2=1,read3=1,update_static=1,update_ttl=1,update_diff1_ts=1,update_diff2_ts=1,update_same1_ts=1,update_same2_ts=1)' cl=ALL n=5000000 -mode cql3 native -rate threads=200 -pop seq=1..5000000
 stress_cmd_complex_verify_delete: cassandra-stress user no-warmup profile=/tmp/complex_schema.yaml  ops'(delete_row=1)' cl=ALL n=500000 -mode cql3 native -rate threads=200 -pop seq=1..500000
+stress_cmd_large_partitions_10m: scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=2000 -clustering-row-count=5555 -clustering-row-size=uniform:1024..2048 -concurrency=5 -connection-count=5 -consistency-level=quorum -rows-per-request=10 -timeout=30s -validate-data
```

Comment:
1. too many partitions - we don't need 2k partitions, 100 or 200 is enough.
2. clustering-row-size=uniform:1024..2048 is too big, we can use 100-1024.
3. Is concurrency 5 enough?
Last, why did you call it 10m? How do you know it finishes in 10 minutes?

Reply:
That was an initial estimation that it'll run for 10 minutes. actually it ran quite more than that - 28 minutes, but all upgrade tests passed ok in the tested job. that means some parameters could indeed be decreased. There was no overload issue as well. 5 concurrency is enough. it is now expected to run for 10 minutes instead of 28.
the command is updated to:
stress_cmd_large_partitions_10m: scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=800 -clustering-row-count=5555 -clustering-row-size=uniform:100..1024 -concurrency=5 -connection-count=5 -consistency-level=quorum -rows-per-request=10 -timeout=30s -validate-data

Reply:
It took 11 minutes to run in recent test 3:
```
< t:2022-08-17 11:17:10,627 f:file_logger.py  l:101  c:sdcm.sct_events.file_logger p:INFO  > 2022-08-17 11:17:10.624: (ScyllaBenchEvent Severity.NORMAL) period_type=begin event_id=cef557a9-e815-448e-81bf-7dd32d471c31: node=Node rolling-upgrade--debian-buster-loader-node-a4b6e7b4-0-1 [35.196.231.171 | 10.142.0.99] (seed: False)
< t:2022-08-17 11:28:07,890 f:file_logger.py  l:101  c:sdcm.sct_events.file_logger p:INFO  > 2022-08-17 11:28:07.887: (ScyllaBenchEvent Severity.NORMAL) period_type=end event_id=cef557a9-e815-448e-81bf-7dd32d471c31 duration=10m57s: node=Node rolling-upgrade--debian-buster-loader-node-a4b6e7b4-0-1 [35.196.231.171 | 10.142.0.99] (seed: False)
```

Reply:
Why don't you use a duration parameter?

Reply:
Because using `duration` doesn't verify the amount of written data. and it may also not allow running a read verification stress afterwards.